png_cicp_chunk_proposal.md

```diff
@@ -0,0 +1,50 @@
+# Adding support for HDR imagery to the PNG format
+Authors (alphabetical): Chris Blume, Chris Seeger, Pierre-Anthony Lemieux
+
+Status: Draft
+
+## Problem to be solved
+The gAMA chunk of the Portable Network Graphics (PNG) format (specified in [PNG]) parameterized the transfer function of the image as a power law. As such, it cannot model the Reference PQ or HLG OTFs specified in [BT2100-1], which are commonly used for HDR images.
+
+This specification uses the existing iCCP chunk to unambiguously signal the color system of an image that uses the Reference PQ EOTF specified in [BT2100-1]. It also allows graceful processing by decoders that do not conform to this specification by recommending fallback values for the gAMA chunk, cHRM chunk, and embedded ICC profile.
+
+An existing W3C group note (https://www.w3.org/TR/png-hdr-pq/) specifies an approach which is limited: it supports signaling of only the ITU BT.2100 PQ EOTF and uses magic values in the iCCP chunk to signal color spaces.
```

Comment:
I guess "magic values" is sufficient to indicate that this is a really bad idea. Specifically, the ICC profile, if applied, does not give the correct rendering and must be ignored :) and only its presence is used as a verbose way to label the content. This proposal is thus a significant improvement on that, in terms of internal consistency and also filesize.